Telemetry/CorePing.swift

```diff
@@ -64,6 +69,13 @@ class CorePing: TelemetryPing {
 
         let timezoneOffset = NSTimeZone.localTimeZone().secondsFromGMT / 60
 
+        let usageCount = UsageTelemetry.getCount(prefs)
+        let usageTime = UsageTelemetry.getTime(prefs)
+        UsageTelemetry.reset(prefs)
+
+        profile.prefs.setInt(Int32(usageTime), forKey: PrefKeyUsageTime)
+        profile.prefs.setInt(Int32(usageCount), forKey: PrefKeyUsageTime)
```

Comment:
Er, thanks. Leftovers from the previous version when this was still in `CorePing.swift`.